### PR TITLE
split campaign and geometry

### LINKED ISSUE #325: Split campaign into 2 files
# Problem

At the moment we have one campaign file. This contains campaign data as well as the geometry. It would be great to view the geometry as a GeoJSON file. 

# Proposed Solution

Lets have 2 files describing one campaign:
- ffffffffffffffff.geojson for the geometry 
- ffffffffffffffff.json for the campaign data

### PR DESCRIPTION
**Pull request by meomancer**
This fix #325 

For execute script to split it, do
- `make shell`
- `python manage.py generate_geometry`

before:
```
/Development/field-campaigner-data/campaign$ ls
20abaee407f243bd94fdde57faedf210.json  57736660413f44b0bad131a82f739583.json  800c9e4c747f49dc8263c00ef1d194b0.json  cb40fabb22134a8891d7f0f87a168612.json  f9b6b31b39204ebca9d6422d2cb167b6.json
217742f498da47d0ab2f0932b523980f.json  5c46420cede4420a847b22cc4109bf1b.json  9e0bf7c14b3d42769c831ad9567fd6b3.json  cdaa6a2fcc53474b966b963e6033619d.json
220a4e8681f9448993a1ffc6836b0245.json  64efc9bea80e4ee5bee420d3f1a1b509.json  afe17248406a463884c82b9008b97344.json  d72c2d29cf844f6ab3e8db9179cffe82.json
541caf2d6a064f0cafc91b71d31468bf.json  78a6d191ed604a15a0b605571d54dfd7.json  b1c270f15bfd48e59543bb09086da0fd.json  e47f8ec4b01147499690707e625cd91e.json
```

after:
```
/Development/field-campaigner-data/campaign$ ls
20abaee407f243bd94fdde57faedf210.geojson  57736660413f44b0bad131a82f739583.json     9e0bf7c14b3d42769c831ad9567fd6b3.geojson  cdaa6a2fcc53474b966b963e6033619d.json
20abaee407f243bd94fdde57faedf210.json     5c46420cede4420a847b22cc4109bf1b.geojson  9e0bf7c14b3d42769c831ad9567fd6b3.json     d72c2d29cf844f6ab3e8db9179cffe82.geojson
217742f498da47d0ab2f0932b523980f.geojson  5c46420cede4420a847b22cc4109bf1b.json     afe17248406a463884c82b9008b97344.geojson  d72c2d29cf844f6ab3e8db9179cffe82.json
217742f498da47d0ab2f0932b523980f.json     64efc9bea80e4ee5bee420d3f1a1b509.geojson  afe17248406a463884c82b9008b97344.json     e47f8ec4b01147499690707e625cd91e.geojson
220a4e8681f9448993a1ffc6836b0245.geojson  64efc9bea80e4ee5bee420d3f1a1b509.json     b1c270f15bfd48e59543bb09086da0fd.geojson  e47f8ec4b01147499690707e625cd91e.json
220a4e8681f9448993a1ffc6836b0245.json     78a6d191ed604a15a0b605571d54dfd7.geojson  b1c270f15bfd48e59543bb09086da0fd.json     f9b6b31b39204ebca9d6422d2cb167b6.geojson
541caf2d6a064f0cafc91b71d31468bf.geojson  78a6d191ed604a15a0b605571d54dfd7.json     cb40fabb22134a8891d7f0f87a168612.geojson  f9b6b31b39204ebca9d6422d2cb167b6.json
541caf2d6a064f0cafc91b71d31468bf.json     800c9e4c747f49dc8263c00ef1d194b0.geojson  cb40fabb22134a8891d7f0f87a168612.json     thumbnail
57736660413f44b0bad131a82f739583.geojson  800c9e4c747f49dc8263c00ef1d194b0.json     cdaa6a2fcc53474b966b963e6033619d.geojson
```

### CODE REVIEW

**Comment by dimasciput** [comment]
@meomancer can you create makefile command instead?

**Comment by cchristelis** [comment]
@meomancer, we need a check to avoid the case that we run the same script twice and end up blank geometries.

File: flask_project/manage.py [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 
-migrate = Migrate(osm_app, db)
 manager = Manager(osm_app)
 
-manager.add_command('db', MigrateCommand)
+@manager.command
+def generate_geometry():
+    generate_geometry_script()
 
+manager.add_command('db', MigrateCommand)
```
Comment (dimasciput) [inline flask_project/manage.py]: I think we can remove this too

**Comment by meomancer** [comment]
hi @cchristelis 
the checker is already there, just generate the campaign that doesn't has geojson yet
https://github.com/hotosm/field-campaigner/pull/339/files#diff-b5bf1098628509314df4862bdf3b4d5eR8

**Comment by meomancer** [comment]
i added `generate-geojson` on Makefile @dimasciput

**Pull request merged**